script/fetch-fixtures

```diff
@@ -11,14 +11,8 @@ fetch_grammar() {
   echo "Updating ${grammar} grammar..."
 
   if [ ! -d $grammar_dir ]; then
-    git clone $grammar_url $grammar_dir --depth=1
+    git clone -b $ref $grammar_url $grammar_dir --depth=1
   fi
-
-  (
-    cd $grammar_dir
-    git fetch origin $ref --depth=1
-    git reset --hard FETCH_HEAD
-  )
```

Comment:
If the grammar repo has already been downloaded, this would update it to the given ref. It seems like that will no longer happen. Am I missing something?